sage/data_manager.py

```diff
@@ -159,7 +159,7 @@ def _should_include(self, file_path: str) -> bool:
         _, extension = os.path.splitext(file_path)
         extension = extension.lower()
         file_name = os.path.basename(file_path)
-        dirs = os.path.dirname(file_path).split("/")
+        dirs = os.path.dirname(file_path).split(os.sep)
```

Comment:
That's a very good idea. For consistency, would you mind making this update in other places in the codebase?

```
grep -nr '.split("/")' sage
sage/sage/retriever.py:244:                items = path.split("/")
sage/sage/retriever.py:256:        root_path = "/" + repo_metadata[0]["file_path"].split("/")[0]
sage/sage/retriever.py:278:        org_name, repo_name = repo_id.split("/") <--- this one shouldn't be updated
sage/sage/retriever.py:279:        items = filename.split("/")
sage/sage/data_manager.py:166:        dirs = os.path.dirname(file_path).split("/")
```